### PR TITLE
Task #67553: Refactor Admin User Layout

### PR DESCRIPTION
**Pull request by janglee83**
## Related Tickets
- [Task #67553](https://edu-redmine.sun-asterisk.vn/issues/67553)

## WHAT (optional)
- Refactor code, build two layouts with two controllers and two views

## HOW
- I write another controller and view folder using namspace

## Evidence (Screenshot or Video)
Screenshot 2023-09-05 at 16 21 04
Screenshot 2023-09-05 at 16 22 54
Screenshot 2023-09-05 at 16 23 00

## Notes (Kiến thức tìm hiểu thêm)
- Tách view bằng controller và namespace

### CODE REVIEW

**Comment by thainda-0252** [comment]
okela

**Comment by vinh-nd2002** [comment]
good chóp.

**Comment by janglee83** [comment]
ready

File: config/routes.rb [hunk 1]
```diff
@@ -1,8 +1,14 @@
 Rails.application.routes.draw do
   scope "(:locale)", locale: /en|vi/ do
-    root "static_pages#index"
-    get "/", to: "static_pages#index"
-    get  "/signup",  to: "users#new"
-    resources :users
+    namespace :user do
```
Comment (ngocvt-0484) [inline config/routes.rb]: site user k cần namespace đâu e. thêm /user k cần thiết. coi mặc đinnh là site user đi e

**Comment by janglee83** [comment]
ready

**Pull request merged**